src/pages/authPage/join.js

```diff
@@ -1,3 +1,56 @@
+import { Link } from "react-router-dom";
+import { Navbar } from "../../components/navbar/navbar";
```

Comment:
One suggestion You can import directly Navbar component in App.js

Reply:
fixed it